src/iosource/Manager.cc

```diff
@@ -89,8 +89,17 @@ Manager::~Manager()
 
 	pkt_dumpers.clear();
 
+#ifndef _MSC_VER
+	// There's a bug here with builds on Windows that causes an assertion with debug builds
+	// related to libkqueue returning a zero for the file descriptor. The assert happens
+	// because something else has already closed FD zero by the time we get here, and Windows
+	// doesn't like that very much. We only do this close when shutting down, so it should
+	// be fine to just skip it.
```

Comment:
Part of it is that libkqueue just blindly returns a zero for a file descriptor all the time on Windows, without checking to see that it's in use by anything else. We could check to see if it's valid before closing it, but that may just lead to some other place crashing because it was expecting to be able to close it.

The issue linked in that comment is asking them to find a way to return an actually-valid file descriptor.